app/src/main/java/mod/trindadedev/ui/fragments/events/EventsManagerCreatorFragment.java

```diff
@@ -30,34 +38,39 @@ public class EventsMakerCreator extends Activity {
     private boolean isEdit = false;
     private String lisName;
 
-    private EventsCreatorBinding binding;
+    private FragmentEventsManagerCreatorBinding binding;
 
+    @Nullable
     @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        binding = EventsCreatorBinding.inflate(getLayoutInflater());
-        setContentView(binding.getRoot());
-        if (getIntent().hasExtra("lis_name")) {
-            lisName = getIntent().getStringExtra("lis_name");
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        binding = FragmentEventsManagerCreatorBinding.inflate(inflater, container, false);
+        View view = binding.getRoot();
+
+        if (getArguments() != null && getArguments().containsKey("lis_name")) {
+            lisName = getArguments().getString("lis_name");
             isActivityEvent = lisName.equals("");
         }
-        if (getIntent().hasExtra("event")) {
-            event_name = getIntent().getStringExtra("event");
+        if (getArguments() != null && getArguments().containsKey("event")) {
+            event_name = getArguments().getString("event");
             isEdit = true;
-            _name = getIntent().getStringExtra("_name");
-            _var = getIntent().getStringExtra("_var");
-            _icon = getIntent().getStringExtra("_icon");
-            _desc = getIntent().getStringExtra("_desc");
-            _par = getIntent().getStringExtra("_par");
-            _spec = getIntent().getStringExtra("_spec");
-            _code = getIntent().getStringExtra("_code");
+            _name = getArguments().getString("_name");
+            _var = getArguments().getString("_var");
+            _icon = getArguments().getString("_icon");
+            _desc = getArguments().getString("_desc");
+            _par = getArguments().getString("_par");
+            _spec = getArguments().getString("_spec");
+            _code = getArguments().getString("_code");
         }
```

Comment:
Please get arguments in Fragment#onCreate(Bundle).

Reply:
ok, I'll do that soon, but I don't know the difference